core/src/com/unciv/ui/screens/diplomacyscreen/OfferColumnsTable.kt

```diff
@@ -18,10 +18,12 @@ import com.unciv.ui.screens.basescreen.BaseScreen
 class OfferColumnsTable(
     private val tradeLogic: TradeLogic,
     private val screen: DiplomacyScreen,
-    private val onChange: () -> Unit
+    private val onChange: () -> Unit,
```

Comment:
The OnChange should be the last parameter of the function, since it's a deferred action